blazor/datagrid/detail-template.md

```diff
@@ -426,4 +424,132 @@ In the following sample, you can get the instance of that particular child gr
 
 ```
 
-> [View Sample in GitHub.](https://github.com/SyncfusionExamples/blazor-datagrid-set-instance-for-child-component)
\ No newline at end of file
+> [View Sample in GitHub.](https://github.com/SyncfusionExamples/blazor-datagrid-set-instance-for-child-component)
+
+## Customize detail template icon
+
+The Detail template icon is used to expand or collapse the detail content. You can customize this icon through CSS itself. You can achieve this requirement by overriding the below two CSS styles.
```

Comment:
The Detail template icon is used to expand or collapse the detail content. You can customize this icon through CSS itself. This can be achieved by overriding the two CSS styles listed below.